src/cryptoadvance/specterext/spectrum/config.py

```diff
@@ -0,0 +1,79 @@
+''' A config module contains static configuration '''
+import logging
+import os
+from pathlib import Path
+import datetime
+import secrets
+from flask import current_app as app
+from cryptoadvance.specter.config import _get_bool_env_var
+
+try:
+    # Python 2.7
+    import ConfigParser as configparser
+except ImportError:
+    # Python 3
+    import configparser
+
+logger = logging.getLogger(__name__)
+
+
+class BaseConfig(object):
+    """Base configuration. Does not allow e.g. SECRET_KEY, so redefining here"""
+    USERNAME='admin'
+    SPECTRUM_DATADIR="data" # used for sqlite but also for txs-cache
+
+    # The prepopulated options
+    ELECTRUM_OPTIONS = {
+        "electrum.emzy.de": {
+            "host": "electrum.emzy.de",
```

Comment:
@moritzwietersheim bring your electrum-servers here! 
For inspriration, have a look in this list:
https://1209k.com/bitcoin-eye/ele.php